front/app/containers/Admin/projects/project/nativeSurvey/FormResults/messages.ts

```diff
@@ -108,6 +108,15 @@ export default defineMessages({
     id: 'app.containers.Admin.projects.project.survey.analysis.refresh',
     defaultMessage: '{ count } new responses',
   },
+  regenerate: {
+    id: 'app.containers.Admin.projects.project.survey.analysis.regenerate',
+    defaultMessage: 'Regenerate',
+  },
+  backgroundTaskFailed: {
+    id: 'app.containers.Admin.projects.project.survey.analysis.backgroundTaskFailed',
+    defaultMessage:
+      'There was an error generating the AI summary. Please regenerate the summary below.',
```

Comment:
Would it make sense to remove this little "summary" duplication?
```suggestion
      'There was an error generating the AI summary below. Please regenerate it.',
```

Reply:
Are there any other errors that can happen but cannot be fixed by regenerating? In this case, maybe it would make sense to specify it in the message
```suggestion
      'There was an error generating the AI summary below. Please try to regenerate it. If the issue persists, contact us.',
```